build.gradle

```diff
@@ -71,6 +72,51 @@ subprojects {
     apply plugin: 'idea'
     apply plugin: 'eclipse'
     apply plugin: 'java'
+    apply plugin: 'com.chrisgahlert.gradle-dcompose-plugin'
+
+    // Using gradle-dcompose-plugin to build/push docker images
+    dcompose {
+        if (rootProject.hasProperty('dockerRegistry')) {
+            def registryName = rootProject.getProperty('dockerRegistry')
+
+            def repositoryName = "${project.name - 'Source' + '-source'}".toLowerCase()
+            if (rootProject.hasProperty('repositoryName')) {
+                repositoryName = rootProject.getProperty('repositoryName')
+            }
+
+            connector {
+                buildFiles = project.copySpec {
+                    from "$projectDir/src/main/docker/"
+                    from "$projectDir/build/distributions/${project.name}.tar"
+                    if (rootProject.hasProperty('connectorSpecificInclusions')) {
```

Comment:
Here, it seems like this could be something that the local project has, too.  For example, some connector could want some standard document included somewhere for and may define that in it's ext props?